code/modules/admin/sql_ban_system.dm

```diff
@@ -317,7 +317,7 @@
 				"}
 				break_counter++
 			output += "</div></div>"
-		var/list/long_job_lists = list("Civilian" = GLOB.civilian_positions,
+		var/list/long_job_lists = list("Civilian" = list("Bartender", "Botanist", "Cook", "Janitor", "Curator", "Lawyer", "Chaplain", "Clown", "Mime", "Assistant", "Gimmick"),
```

Comment:
in before someone forgets to add new jobs to this list

Reply:
Yeah this should definitely be generated dynamically by just making a list from the GLOB minus the gimmick jobs.

Reply:
Gimmicks jobs are not stored in a global list instead are added to the civilian_positions when initialized. I could split them both into their respective global lists and just use civilian_positions + "gimmick".

Reply:
You can just keep a separate list of gimmick roles and then subtract them from this list.